src/style.css

```diff
@@ -2,0 +1,67 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.search-bar {
+  margin-top: 20px;
+}
+
+.search-bar-top {
+  height: 70px;
+  margin-top: 20px;
```

Comment:
Be careful with fixing container size. In this case, you've fixed the height for '.search-bar-top'. Unless it's a specific requirement, it's better to let the content dictate the size of the container to avoid overflow or accidental scroll bar.